.circleci/config.yml

```diff
@@ -85,6 +85,7 @@ jobs:
             docker pull -q lambci/lambda:20191117-python3.6
             docker pull -q lambci/lambda:20191117-dotnetcore2.0
             docker pull -q lambci/lambda:dotnetcore3.1
+            docker pull -q lambci/lambda:dotnet6.0
```

Comment:
waiting https://github.com/lambci/docker-lambda/pull/359 to get merged

Reply:
Hi there! Lambci is pretty dead right now. Until we have the lambda rework in place, it might be a good solution to use the repository of mlupine: https://hub.docker.com/layers/docker-lambda/mlupin/docker-lambda/dotnet6/images/sha256-fc3edf2e50307a5c81e4da7690f6cb6427819279081e1d9b16e52a294bf21738?context=explore , as we do already with python3.9.

Reply:
We would need to adjust the image though, in lambda_executor.py:1596.

Reply:
Finally got this working. 
@dfangl can you take a final review?

Reply:
Please add the docker pull -q ... part back in for the mlupin image!

Reply:
👍🏻 added back